package.json

```diff
@@ -18,7 +18,7 @@
     "test": "grunt eslint fh-test"
   },
   "engines": {
-    "node": ">=0.4.12"
+    "node": ">=4.9.1"
```

Comment:
Was there a need to drop the node version to >=4.9.1

Reply:
All before 4 is deprecated and 4.9.1 is the latest version of 4 which is the only which so far is not deprecated indeed in the NGUI/Studio. What do you think? Could we move forward?

Reply:
Misread 0.4.12 as 4.12 :) yes - makes sense to align with what's used in the studio until we remove 4 from it

Reply:
👍 Cool thanks you for your collaboration and review.